app/models/responsible_body.rb

```diff
@@ -63,6 +72,14 @@ def can_school_be_added_to_virtual_cap_pools?(school)
       !has_school_in_virtual_cap_pools?(school)
   end
 
+  def can_school_be_removed_from_virtual_cap_pools?(school)
```

Comment:
It looks like you followed the naming convention of the method above `can_school_be_added_to_virtual_cap_pools?`, however we could take this opportunity to fix the naming.
[https://rubystyle.guide/#bool-methods-prefix](url)

Reply:
"has" is a grey area though. Let's leave it.

Reply:
👍  Yes, I was following the existing pattern but I have renamed this one and the twin about adding